kolibri/core/logger/models.py

```diff
@@ -194,7 +201,12 @@ def update_log(cls, user):
                 or timezone.now() - user_session_log.last_interaction_timestamp
                 > timedelta(minutes=5)
             ):
-                user_session_log = cls(user=user)
+                parsed_string = user_agent_parser.Parse(user_agent)
+                device_info = parsed_string['os'].get("family", "") + ","           \
+                              + (parsed_string["os"].get("major", "") or "") + "/"  \
+                              + parsed_string['user_agent'].get("family", "") + "," \
+                              + (parsed_string["user_agent"].get("major", "") or "")
```

Comment:
These lines are quite difficult for me to parse - I think using a string template and formatting it would be make this clearer.

I assume this is equivalent to:
```suggestion
                device_info = "{os_family},{os_major}/{browser_family},{browser_major}".format(
                    os_family=parsed_string["os"].get("family", ""),
                    os_major=parsed_string["os"].get("major", ""),
                    browser_family=parsed_string["user_agent"].get("family", ""),
                    browser_major=parsed_string["user_agent"].get("major", "")
                )
```